src/containers/Secrets/Secrets.js

```diff
@@ -55,6 +57,9 @@ const initialState = {
   selectedType: 'password'
 };
 
+const secretCreationHelperString =
+  'Use Password if you plan to have a PipelineRun that clones from a Git repository or image registry that requires authentication. Use Access Token when interacting with webhooks or PullRequest resources. Check the official Tekton Pipelines documentation for more details.';
```

Comment:
This also applies to TaskRuns, and I think 'official' is implied. How about something like this?

> Use Password with git or image PipelineResources that require authentication. Use Access Token with webhooks or with pullRequest PipelineResources. Check the Tekton Pipelines documentation for more details on authentication.